code/randomized_algorithms/src/README.md

```diff
@@ -1,4 +1,34 @@
-# cosmos
-Your personal library of every algorithm and data structure code that you will ever encounter
+# Randomized Algorithms
+A randomized algorithm is that which receives in addition to its input data,a stream of random bits for making random choices.
+For example, even when the input is fixed the execution time of randomized algorithm is a random variable. <br >
+Practically computers can't generate completely random numbers so they are approximated using pseudorandom number generator.
+## Types of Randomized Algorithms
+* Monte Carlo
+* Las Vegas
 
-Collaborative effort by [OpenGenus](https://github.com/opengenus)
\ No newline at end of file
+###### Monte Carlo :
+Monte Carlo may produce incorrect answer but we are able to bound its error by probability.Also, by running it many times on independent random variable, we can make failure probability arbitrarily small at the expense of running time.<br >
```

Comment:
...probability.Also... : missing comma after dot. Look for missing spaces in the entire file please.